prov/efa/src/efa_av.c

```diff
@@ -266,18 +263,12 @@ int efa_conn_rdm_init(struct efa_av *av, struct efa_conn *conn, bool insert_shm_
 	int err, ret;
 	char smr_name[EFA_SHM_NAME_MAX];
 	size_t smr_name_len;
-	struct efa_rdm_ep *efa_rdm_ep;
-	struct efa_rdm_peer *peer;
 
 	assert(av->ep_type == FI_EP_RDM);
 	assert(conn->ep_addr);
 
-	/* currently multiple EP bind to same av is not supported */
 	assert(!dlist_empty(&av->util_av.ep_list));
```

Comment:
do you need such assert now since av and peer is decoupled?

Reply:
That means applications can do av insert before the AV is bound to any endpoints. Is that legal? Can't figure out from the documentation.

Reply:
Yeah, I don't know why it is not legal, and docs doesn't say it's illegal. You should be able to insert an address to the av as long as it's a valid ep name